tests/common.py

```diff
@@ -392,45 +394,59 @@ def assert_equal_vectors(self, truth, test, rtol):
             assert 0
 
     def compare_forces(
-        self, x, params, box, lamb, ref_potential, test_potential, rtol, precision, atol=1e-8, benchmark=False
+        self,
+        x: NDArray,
+        params: NDArray,
+        box: NDArray,
+        lambdas: Union[List[float], float],
```

Comment:
I can revert this change if we want as it is a bit strange to take a list or a single value, but it makes things a lot faster where we want to test multiple lambda values.

Reply:
nit: for simplicity, would it make sense to only accept a list and update the call sites?

Reply:
(this might also nudge future uses away from running this in a loop over lambda)

Reply:
Resolved in [605abd8](https://github.com/proteneer/timemachine/pull/699/commits/605abd81941bce36592623cb0e033b964b29ac4e)